index.html

```diff
@@ -1280,6 +1270,158 @@ <h3>
           conventions or limitations of the host operating system.
         </p>
       </section>
+      <section>
+        <h3>
+          `*_localized` members
+        </h3>
+        <p>
+          A <dfn>localizable member</dfn> is a [=manifest=] member that can be
+          localized. Each [=localizable member=] of the [=manifest=] has a
+          corresponding <dfn>localized member</dfn> with the suffix
```

Comment:
This is really hard to read, in that `localizable member` and `localized member` are super similar looking. Consider separating the terms somewhat more?

> A `localizable member` is a _manifest_ member that can be localized.
> This can include translation or other locale-based customization.
> When present, the localized representations of a `localizable member`
> appear in a corresponding member with the suffix `_localized`. ...

Reply:
Resolution: `<dfn>`*_localized` member</dfn>`